doaj/DOAJ/DOAJRequest.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python3
+
+#author:  Víctor Moreno Marín, Gerardo Gutiérrez
+#email: vmorenomarin@gmail.com, muzgash@gmail.com, 
+
+from pymongo import MongoClient
+from bson.objectid import ObjectId
+from forex_python.converter import CurrencyRates
+import requests             
+import json   
+import numpy as np  
+
+
+class DOAJRequest:
+    '''
+    Create a DOAJ object to request and save data from DOAJ site using its API.
+    '''
+
+    def __init__(self,database_name,collection):
+        self.client = MongoClient()
+        self.db = self.client[database_name]
+        self.collection = self.db[collection]
+     
+    def getIssnLists(self): 
+        '''
+        Returns print and electronic ISSN's from Lens collection.
+        '''
+        pissn_list = []
+        eissn_list = []
+        for register in self.collection.find():
+            if "source" in register.keys():
+                if register["source"]:
+                    if not "issn" in register["source"]:
+                        continue
+                    for issn in register["source"]["issn"]:
+                        if issn["type"] == "print":
+                            pissn_list.append(issn["value"])
+                        elif issn["type"] == "electronic":
+                            eissn_list.append(issn["value"])
+
+        return pissn_list, eissn_list
+
+    def getAPC(self,issn_list=[]):      
```

Comment:
the idea of this method is to save those issns in mongo,
you need to create a mongo database called DOAJ and in a collections called stage you should save all the data provide for DOAJ not only the APC, even if the given ISSN does not have that information.